v22.2/known-limitations.md

```diff
@@ -10,6 +10,85 @@ This page describes newly identified limitations in the CockroachDB {{page.relea
 
 ## New limitations
 
+### DROP OWNED BY does not support drop functions
+
+`DROP OWNED BY` drops all owned objects as well as any grants on objects not owned by the role.
+
+In its current implementation, this statement does not drop functions. Users must drop their functions manually.
+
+[Tracking GitHub Issue](https://github.com/cockroachdb/cockroach/issues/90476)
+
+{% comment %}
+    initial DROP OWNED BY implementation PR/issue:
+    https://github.com/cockroachdb/cockroach/pull/82936
+    https://github.com/cockroachdb/cockroach/issues/55381
+    doc work in progress (to link to after merge):
+    https://github.com/cockroachdb/docs/pull/15094/files
+    https://cockroachlabs.atlassian.net/browse/DOC-4770 
+    review: rafiss (or devadvocado)
+{% endcomment %}
+
+### DROP OWNED BY not supported where role has synthetic privileges
```

Comment:
@rmloveland also added this in https://github.com/cockroachdb/docs/pull/15094

Reply:
@rafiss thanks for the ping

@mikeCRL since this is for the v22.2.0 release and it touches multiple feature areas I can wait for this one to merge and then update my PR on top of it

Sound good?

Reply:
Thanks, Rafi.
@rmloveland Do you think I can simply replace the text under this heading with the include here https://github.com/cockroachdb/docs/pull/15094/files#diff-fe6e9708e2d77f8f1489e7bdee6434fa149a3f3887fae95935b0e5ec0fe5fdce , or what would you recommend?

Reply:
> @rmloveland Do you expect that PR to be merged by EOD Monday Dec 5? 

Probably?  but this PR is more important than that one.  I wouldn't block this PR on that one.  IMO it would be better to merge this one and then I can push that one after

> If so: Do you think I can simply replace the text under this heading with the [include](https://github.com/cockroachdb/docs/pull/15094/files#diff-fe6e9708e2d77f8f1489e7bdee6434fa149a3f3887fae95935b0e5ec0fe5fdce) there, or do you have any recommendation otherwise?

This is ~basically what I'm going to do after this one goes in

Reply:
@rmloveland Apologies, I had loaded the PR just before you made your first comment—I hadn't seen it when I left my comment. Getting this PR out first sounds good.

But I just want to ensure I understand: sounds good to you if I were to copy your text and use it here, and then you'll update your PR to replace my text with the include reference? That works for me—please just correct me if I'm misinterpreting. Thanks!

Reply:
feel free to copy whatever you want from my PR, or leave this (presumably already reviewed) text as-is if that's faster right now

Reply:
Decided to use @rmloveland's superior language from the linked PR. (Thanks!) Updated this PR. No additional review is needed.

Also added a [comment](https://github.com/cockroachdb/docs/pull/15094/files#r1039812161) on Rich's PR about a couple follow-up items.

Reply:
@rafiss @rmloveland @abhinavg6 It looks like the latest `GRANT` doc refers only to "global privileges"—should I rename "synthetic privileges" to "global privileges" in this heading? Is the former an internal-only term?

Reply:
thanks @andf-crl 

this Jira issue may be relevant to your interests re: "synthetic" privileges and the docs: https://cockroachlabs.atlassian.net/browse/DOC-6325